api_server/Cargo.toml

```diff
@@ -26,3 +26,4 @@ kernel = { path = "../kernel" }
 memory_model = { path = "../memory_model" }
 net_util = { path = "../net_util" }
 rate_limiter = { path = "../rate_limiter" }
+dumbo = { path = "../dumbo" }
```

Comment:
It's nice to have the dependencies listed in alphabetical order. Also, is there any compelling reason to move `MacAddr` from `net_util` to `dumbo`? I think `net_util` is likelier to end up as a public crate first, and it now depends on `dumbo`, which makes things a bit awkward in that regard.

Reply:
ah you have this point, now there isn't, but back then when I was editing the code I guess dumbo depended on `net_util` in some tests which made a circular dependency, I'll restore `MacAddr` there. 
I have one though, doesn't keeping `MacAddr` in `dumbo` makes it look like a single logical block containing all network-stack-related functionality?

Reply:
just for documentation; the circular dependency occurs because [previously](https://github.com/firecracker-microvm/firecracker/pull/1238/files#diff-505a782d905cdaf207a79b70c1d56610R241), the tests used network structs from `pnet` and now it switched to `dumbo`'s. `dumbo` needs `MacAddr` for `ns` (and hence the circular dependency will happen if `MacAddr` is in `net_util`)